docs/how-to-guides/advanced-user-guides/istio-setup-guide/generate-and-view-traffic.md

```diff
@@ -6,19 +6,21 @@ title: Generate and View Traffic from Istio
   <link rel="canonical" href="https://ranchermanager.docs.rancher.com/how-to-guides/advanced-user-guides/istio-setup-guide/generate-and-view-traffic"/>
 </head>
 
-This section describes how to view the traffic that is being managed by Istio.
+## Prerequisites
 
-## The Kiali Traffic Graph
+To enable traffic to show up in the graph, ensure you have prometheus installed in the cluster. `Rancher-istio` installs Kiali configured by default to work with the `rancher-monitoring` chart. You can use `rancher-monitoring` or install your own monitoring solution.
```

Comment:
```suggestion
To enable traffic to show up in the graph, ensure that you have Prometheus installed in the cluster. `Rancher-istio` installs Kiali, and configures it by default to work with the `rancher-monitoring` chart. You can use `rancher-monitoring` or install your own monitoring solution.
```

Realize that this is the original text but needed to edit

Reply:
Applied suggestion